pcmanfm/settings.cpp

```diff
@@ -439,6 +439,33 @@ bool Settings::saveFile(QString filePath) {
     return true;
 }
 
+QList<int> Settings::iconSizes(IconType type) const {
+    QList<int> sizes;
+    switch(type) {
+    case Big:
+        sizes << 96 << 72 << 64 << 48 << 32;
```

Comment:
How about using `constexpr`'s (and return the `const QList<int> &` here?

Reply:
No objection. Would you please add a commit?

Reply:
OK... the `QList` can't be `constexpr`, but also the `static const` should perform better... pushed the commit.

Reply:
Yes, I saw it. It's good. Thanks!